datastore/datastore.go

```diff
@@ -93,15 +155,26 @@ func (self ElasticDatastore) SetSubject(
 		return err
 	}
 
+	id := uuid.New()
```

Comment:
You can not use a random number for the downloads because the gui wont be able to find them - the id is set to be a reproducible hash of the vfs path so both the ingestor and the gui will arrive at the same hash.